ocs_ci/utility/users.py

```diff
@@ -0,0 +1,148 @@
+import os
+import random
+import string
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import exec_cmd
+
+
+def add_htpasswd_user(username, password, htpasswd_path):
+    """
+    Create a new user credentials with provided username and password.
+    These will be saved in file located on htpasswd_path. The file will
+    be created if it doesn't exist.
+
+    Args:
+        username (str): Name of a new user
+        password (str): Password for a new user
+        htpasswd_path (str): Path to httpasswd file
+
+    """
+    if os.path.isfile(htpasswd_path):
+        cmd = ['htpasswd', '-B', '-b', htpasswd_path, username, password]
+    else:
+        cmd = ['htpasswd', '-c', '-B', '-b', htpasswd_path, username, password]
+    exec_cmd(cmd)
```

Comment:
Do we want to mask the password in the log? We have secrets parameter for this purpose:
https://github.com/red-hat-storage/ocs-ci/blob/c1abe2b69af1b4aabc7e04aa7e5f8c4939a094f5/ocs_ci/utility/utils.py#L419

And this is proper place where to use it I guess.